tests/cluster-check.sh

```diff
@@ -91,22 +95,26 @@ EOF
     qsub ./job2.sh
 
 elif test "$scheduler" = "torque" ; then
+    _chost=$(pbsnodes -l free | head -n 1 | cut -d ' ' -f1)
```

Comment:
If there aren't yet free compute nodes the `_chost` will be an empty value because `pbsnodes -l free` doesn't return any host.
Then in the next command the `_ppn` will be the value of the `np` of the master host or a multiple row value if there are other (not free) hosts. 
E.g.:
```
$ _ppn=$(pbsnodes | tr -d '\t ' | sed -n '/np=/{s/^np=\([0-9]\+\)/\1/;p;}')
1000
4
4
```
Then the following submission will fail:
```
$ qsub -l nodes=1:ppn=${ppn} ./job2.sh
qsub: script file '4' cannot be loaded - No such file or directory
```

We could check the `ppn` value and fail with a more clear message or perform a loop waiting for the free compute nodes.

Reply:
Improved the way we retrieve the slots per node for the rare cases (seen in our tests) where the first compute node was not there when trying to submit the jobs.